### DISCUSSION TITLE
Markdown or Content Component

### DISCUSSION THREAD

**Discussion by jsjoeio** [resend/react-email#695] [Ideas]
### Goals

1. Let me write my email in plain text but render it pretty in HTML

### Background

Sometimes I have the content in plain text and all I want to do is paste it inside a component and have `react-email` "beautify" it for me.

Astro has a `Content` component: [link](https://docs.astro.build/en/guides/markdown-content/#the-content-component)

Tailwind has their [typography plugin](https://tailwindcss.com/docs/typography-plugin)

### Proposal

Usage might be like this: 

```tsx
import { Content } from "@react-email/components";

const text = `Hi there,

Thanks for signing up for my product! Here are some things you can do with it:
1. use it 
2. download it
3. look at it

Let me know what you think!

Yours truly,
Joe`

export const EmailTemplate = () => (
  <Content>{text}</Content/>
)
```

And under the hood, it would wrap each paragraph in `<Paragraph />` and `<OL />` etc.

Maybe

**Comment by bukinoshita** [comment]
Closing this in favor of https://github.com/resendlabs/react-email/discussions/573
